test/server/admin/BUILD

```diff
@@ -79,7 +79,6 @@ envoy_cc_test(
         "//test/mocks/server:admin_stream_mocks",
         "//test/test_common:logging_lib",
         "//test/test_common:real_threads_test_helper_lib",
-        "//test/test_common:test_runtime_lib",
```

Comment:
This dep can also be removed in the other targets here that we removed the include.